tests/Tests/500__jupyterhub/test-jupyterlab-time.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             DateTime
+Library             OpenShiftCLI
+Library             DebugLibrary
+
+Suite Setup         Load Spawner Page
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+@{LIST_OF_IMAGES}       s2i-minimal-notebook    s2i-generic-data-science-notebook
+...                     pytorch                 tensorflow    minimal-gpu
+
+
+*** Test Cases ***
+Average Time For Spawning
+    [Documentation]    Calculates avg time taken by server to start
+    [Tags]    Tag
+    ${total_avg} =    Set Variable
+    FOR    ${image}    IN    @{LIST_OF_IMAGES}
+        ${avg} =    Set Variable
+        FOR    ${counter}    IN RANGE    4
+            ${sum} =    Spawn and Stop Server    ${image}
+            ${avg} =    Evaluate    ${avg} + ${sum}
+        END
+        ${avg} =    Evaluate    ${avg}/4
```

Comment:
You should do range of 5 and exclude the first attempt from the average, as that might be longer because the image is being downloaded to the node

Reply:
Done